Labyrinth/src/Physics/BoardDisplayer.java

```diff
@@ -68,7 +68,7 @@ private void paintTiles(Graphics g){
 	public void paintCharacters(Graphics g){
 		for (Player c : board.players){
 			GuiPlayer gc = new GuiPlayer(c);
-			Point[] tlBr = screenBounds(c.x, c.y);
+			Point[] tlBr = screenBounds(c.location().x, c.location().y);
```

Comment:
We should change this method to just use c.location(), or make a wrapper method that calls `screenBounds(p.x, p.y)`.